core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -97,7 +99,8 @@ var ExplorationEditorSettingsTab = function() {
     await waitFor.visibilityOf(
       explorationSummaryTile, 'Summary Tile takes too long to appear');
     expect(await explorationSummaryTile.isPresent()).toBeTruthy();
```

Comment:
This can be removed because it may never fail here. We already wait for the visibility of `explorationSummaryTile` in the previous line so if the summary tile were not present the previous statement would time out and report an error.